alyssa_mckee/Django/restful_users/apps/users/views.py

```diff
@@ -0,0 +1,76 @@
+from django.shortcuts import render, redirect, reverse
+from django.contrib import messages
+from .models import User
+
+# Create your views here.
+def index(req):
+	users = User.objects.all()
+	context = {"users":users}
+	return render(req, "users/index.html", context)
+
+def new(req):
+	#add to database
+	context = {}
+	if 'errors'in req.session:
```

Comment:
Very cool! This is definitely a great functionality to have to improve a user's experience inside your website! +2